google/resource_google_project_default_service_accounts_test.go

```diff
@@ -0,0 +1,4 @@
+package google
+
+// IDK if I should do TDD for terraform providers development
+// Do you guys do?
```

Comment:
It's up to you whether you want to write the tests first or write the tests after you've written the code, but we'll definitely need to see working acceptance tests in order to merge the resource.

Reply:
sure. I was more interested in knowing if you guys follow TDD or not when coding terraform providers.